### ISSUE TITLE
Handle old packages deprecated licences property

### ISSUE THREAD

**Issue by alessioalex** [robertkowalski/get-license-npm#2]
According to the [npm documentation](https://docs.npmjs.com/files/package.json) (and from experience) [some modules](https://github.com/adamwdraper/Numeral-js/blob/master/package.json#L30-L34) use a `licenses` property where they specify an array of licenses.

While that style is indeed deprecated, it would be a nice to have for this module to extract the license info.

Additionally, if they don't include a license(s) property at all but do have a license file I was thinking it might be a good option to try to extract the license type from it (MIT, Apache 2, etc).

@robertkowalski let me know if you'd like my help on this task, or any other. Thanks

**Comment by robertkowalski** [comment]
i agree with you. for my use case it covered most cases, but we can do a lot better.

>  let me know if you'd like my help on this task, or any other

any help is welcome!
